series/supportedseries_linux_test.go

```diff
@@ -104,7 +104,7 @@ func (s *supportedSeriesSuite) TestSupportedJujuControllerSeries(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	s.PatchValue(series.UbuntuDistroInfoPath, filename)
 
-	expectedSeries := []string{"focal", "bionic", "xenial"}
+	expectedSeries := []string{"groovy", "focal", "bionic", "xenial"}
```

Comment:
why is groovy added to this one?  It look like a LTS list.

Reply:
This is just Supported vs SupportedLts